content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -1,50 +1,169 @@
-- Cloud instances take some time to provision ([SLA](../../../cloud/index.md#slas-for-managed-instances)), that makes it so signing up for a cloud instance is not instant.
+- Cloud instances take some time to provision ([SLA](../../../cloud/index.md#slas-for-managed-instances)), that makes it so signing up for a cloud instance is not instant. To work around this limitation, we pre-provision instances in some cases, with the trade-off that they have random subdomains (eg. `hjbkn.sourcegraph.com`).
 - The workflow for qualifying a user that requests a trial instance, provisioning the instance, handing it off to the requesting user, and helping them onboard is very manual. This last bit will be automated in the short term (next few months).
+- The onboarding on Sourcegraph instances is very rudimentary. The Product Growth team is working on that next.
+- We aim to give all self-serve trials a white-glove experience through custom onboarding emails, sessions with Customer Engineers, etc. Most of this will eventually be automated.
 
-## Expectations from stakeholders
+## Workflow
+
+Here is an overview of the workflow and the role of any and all Sourcegraph teams.
+
+TLDR:
+
+- We maintain a list of prequalified target accounts, that will instantly get an instance in-product
+- All others go through manual qualification by inbound SDRs, and if qualified, an instance is provisioned by the Cloud team and handed-off to the reqester by a CE
+- Instance hand-off (if not automated), white-glove onboarding, and trial extension/termination/conversion are owned by the CE assigned to the Opportunity.
+- Trials should be terminated after 15 days if there's no activity on the instance, and extented to 30 days if there's any activity.
+
+![Workflow](https://storage.googleapis.com/sourcegraph-assets/growth/flow-v2.svg)
+(You can also view this flow [here](https://app.excalidraw.com/l/4Dr1S6qmmY7/4S6Sc9L2aDO))
+
+Now let's dive into the details of the workflow. CE or AE actions are highlighted by 🟢, inbound SDR actions by 🟡, and named SDR actions by 🟠.
+
+Assume `bob@acme-corp.com` signs up for an instance on signup.sourcegraph.com.
+
+### First step: signup, qualification and instance delivery
+
+#### Case 1: acme-corp.com already has a cloud instance
+
+If there is already a cloud instance owned by `acme-corp`, which is defined as a cloud instance whose initial admin has an `@acme-corp.com` email, then:
+
+1. `bob` is automatically redirected to that instance.
+1. a notification is sent in #cloud-trial-alerts, powered by this [zap](https://zapier.com/editor/167443639/published)
+1. 🟢 the AE/CE should reach out to the instance admin and user to understand if they should be added to the instance
```

Comment:
Should we trust the AE/CE to use judgement here or prescribe something more? I would expect some confusion on their part (who do I reach out to first, do I reach out in the same email, etc) especially because we can't tell whether a user has set up SSO or not

Reply:
I've found that AEs / CEs know how to do this (it's happened a few times) and have a feel for the preference of their champion.
Do you have a good default policy in mind? I'd be down for adding that